src/ui/numeralFormat.ts

```diff
@@ -136,14 +140,11 @@ class NumeralFormatter {
     }
 
     formatShares(n: number): string {
-        if (n < 1000) {
```

Comment:
Removing this is incorrect. Shares are always whole number.

Reply:
I fixed that bad boy